namespaces/cloud-platform-test-1.k8s.integration.dsd.io/traefik/traefik-deployment.yaml

```diff
@@ -42,7 +42,10 @@ kind: Service
 apiVersion: v1
 metadata:
   name: traefik-ingress-service
-  namespace: kube-system
+  namespace: ingress-controllers
+  annotations:
+      # Have `external-dns` create a Route53 record pointing to this ELB
+      external-dns.alpha.kubernetes.io/hostname: "*.test-1-traefik.k8s.integration.dsd.io"
```

Comment:
I don't think `external-dns` can set this record? See https://github.com/ministryofjustice/kubernetes-investigations/blob/master/cluster-components/external-dns/cloud-platform-live-0-helm-values.yaml#L17-L18